signalfx-yammer/src/main/java/com/signalfx/codahale/reporter/SfUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.signalfx.codahale.reporter;
+
+import com.yammer.metrics.core.Gauge;
```

Comment:
More duplicate of https://github.com/signalfx/signalfx-java/blob/master/signalfx-codahale/src/main/java/com/signalfx/codahale/reporter/SfUtil.java